src/meta/src/manager/cluster.rs

```diff
@@ -347,16 +360,17 @@ where
         &self,
         worker_type: WorkerType,
         worker_states: Option<Vec<State>>,
+        list_unschedulable: bool,
```

Comment:
Should always be list_schedulabel. Try not to confuse this with list_unschedulable